meeshkan/serve/mock/faker/stateless_faker.py

```diff
@@ -0,0 +1,323 @@
+import json
+import random
+import typing
+from dataclasses import dataclass
+from functools import reduce
+from typing import Any, Mapping, Sequence, Union, cast
+
+from http_types import Request, Response
+from openapi_typed_2 import OpenAPIObject, Reference, Schema, convert_from_openapi
+
+from meeshkan.serve.mock.faker.faker_base import FakerBase
+from meeshkan.serve.mock.faker.faker_exception import FakerException
+from meeshkan.serve.mock.matcher import (
+    change_ref,
+    change_refs,
+    get_response_from_ref,
+    ref_name,
+)
+from meeshkan.serve.mock.storage.entity import Entity
+from meeshkan.serve.mock.storage.mock_data import MockData
+from meeshkan.serve.utils.opanapi_utils import get_x
+
+
+@dataclass
+class FakerData:
+    spec: OpenAPIObject
+    path_item: str
+    schema: typing.Dict
+    request: Request
+    entity: typing.Optional[Entity]
+    storage: MockData
+    updated_data: typing.Any
+
+
+class StatelessFaker(FakerBase):
+    _LO = -99999999
+    _HI = 99999999
+
+    responses_error = (
+        "While a stub for a specification exists for this endpoint, it contains no responses. "
+        "That usually means the schema is corrupt or it has been constrained too much "
+        "(ie asking for a 201 response when it only has 200 and 400)."
+    )
+
+    # to prevent too-nested objects
+    def sane_depth(self, n):
+        return max([0, 3 - n])
+
+    def fake_object(self, faker_data: FakerData, schema: Any, depth: int) -> Any:
+        addls = (
+            {}
+            if "additionalProperties" not in schema
+            else {
+                k: v
+                for k, v in [
+                    (
+                        self._text_faker.name(),
```

Comment:
I think it's this class's business to decide which faker to use, not the parent's?

Reply:
I moved text_faker to StatelessFaker if you mean that